taskwiki/regexp.py

```diff
@@ -44,47 +44,47 @@
+        r'(?!\|)'                        # (But not two, that would be a preset)
+        r'(?P<filter>[^=\|]*?)'          # Filter
+        r'('                             # Optional defaults
+          r'\|'                          # Bar
+          r'(?P<defaults>[^=\|]+?)'      # Default attrs
         r')?'
-        r'\s*'                       # Any whitespace
-        r'(#(?P<source>[A-Z]))?'     # Optional source indicator
-        r'\s*'                       # Any whitespace
-        r'(\$(?P<sort>[A-Z]))?'      # Optional sort indicator
-        r'\s*'                       # Any whitespace
-        r'(\*(?P<count>[0-9]+))?'    # Optional count indicator
-        r'\s*'                       # Any whitespace
-        r'[=]+'                      # Header ending
+        r'\s*'                           # Any whitespace
+        r'(#(?P<source>[A-Z]))?'         # Optional source indicator
+        r'\s*'                           # Any whitespace
+        r'(\$(?P<sort>[A-Z]))?'          # Optional sort indicator
+        r'\s*'                           # Any whitespace
+        r'(limit:(?P<count>[0-9]+))?'    # Optional count indicator
+        r'\s*'                           # Any whitespace
+        r'[=]+'                          # Header ending
     ),
     'markdown':
     re.compile(
-        r'^'                         # Starts at the begging of the line
-        r'(?P<header_start>[#]+)'    # Heading begging
-        r'(?P<name>[^#\|\[\{]*)'     # Name of the viewport, all before the | sign
-                                     # Cannot include '[', '#', '|', and '{'
-        r'\|'                        # Bar
-        r'(?!\|)'                    # (But not two, that would be a preset)
-        r'(?P<filter>[^#\|]*?)'      # Filter
-        r'('                         # Optional defaults
-          r'\|'                      # Bar
-          r'(?P<defaults>[^#\|]+?)'  # Default attrs
+        r'^'                             # Starts at the begging of the line
+        r'(?P<header_start>[#]+)'        # Heading begging
+        r'(?P<name>[^#\|\[\{]*)'         # Name of the viewport, all before the | sign
+                                         # Cannot include '[', '#', '|', and '{'
+        r'\|'                            # Bar
+        r'(?!\|)'                        # (But not two, that would be a preset)
+        r'(?P<filter>[^#\|]*?)'          # Filter
+        r'('                             # Optional defaults
+          r'\|'                          # Bar
+          r'(?P<defaults>[^#\|]+?)'      # Default attrs
         r')?'
-        r'\s*'                       # Any whitespace
-        r'(#(?P<source>[A-Z]))?'     # Optional source indicator
-        r'\s*'                       # Any whitespace
-        r'(\$(?P<sort>[A-Z]))?'      # Optional sort indicator
-        r'\s*'                       # Any whitespace
-        r'(\*(?P<count>[0-9]+))?'    # Optional count indicator
```

Comment:
This looks good, but it seems to add a lot of unrelated whitespace changes. Are these necessary? I'm a bit confused whether this reformatting is on purpose.. it seems to add couple of spaces, but breaks consistency?

Reply:
I fixes some inconsistent alignment. I moved all comments a bit to the right to have the code breath. The count regex was 6 characters longer that the longest other regex and the most characters any other regex was separated from the comments was 5 characters. This meant I had to move them all to have them align with the new comment for the count regex.

Reply:
I think that's because `r'(limit:(?P<count>[0-9]+))?'` is longer than the rest and the comments wouldn't be aligned.

Reply:
(Yeah, it would be easier to review if the comments were moved in another commit, but I'm not sure that level of pedantry is warranted here. :-))